src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,90 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage{
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    //Web-Elements
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By plusButton = By.cssSelector("#playlists > h1 > i");
+    By playlistNewBtn = By.cssSelector("#playlists > nav > ul > li:nth-child(1)");
+    By inputPlaylist = By.cssSelector("input[placeholder='↵ to save']");
+    By popUpMessage = By.cssSelector("div.success.show");
+    By editBtn = By.xpath("//section[@id='playlists']//nav[@class='menu playlist-item-menu']//li[text()='Edit']");
+    By playlistName = By.cssSelector("input[data-testid='inline-playlist-name-input']");
+    By deletePlaylist = By.xpath("//button[@class='del btn-delete-playlist']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```